### DISCUSSION TITLE
Natural oil spill?

### DISCUSSION THREAD

**Discussion by AndresSepulveda** [OpenDrift/opendrift#1332] [Q&A]
Hi,

In case I want to study the dispersion of a natural source of oil. What option could be selected in openoil?

**Comment by knutfrode** [comment]
By natural oil spill, do you mean a seafloor seep/leakage?
If so, you can use `z='seafloor'` when seeding elements, and chose an oil type from the ADIOS database: https://adios.orr.noaa.gov/oils

Apart from that, I should mention that the process `dispersion` in OpenDrift means removing/deactivating small, entrained oil particles, in accordance with a "historical trick" to deactivate these particles to save computational time.

However, I would suggest rather to not use this process (i.e. `o.set_config('processes:dispersion', False)`) but rather have the other relevant processes (entrainment, vertical mixing, emulsification) activated, like in this example:
https://opendrift.github.io/gallery/example_oil_budget_noaa.html
With this, small, entrained particles are in the state of `submerged` (ref oil budget plots) and may still resurface at a later stage, if wind/waves/turbulence calms down.
